src/App.tsx

```diff
@@ -105,6 +105,7 @@ function App() {
   )
 
   const [currentStreak, setCurrentStreak] = useLocalStorage('current-streak', 0)
+  const [guessesInStreak, setGuessesInStreak] = useLocalStorage('guesses-in-streak', -1)
```

Comment:
Initialize it to -1 here, then when they start their next streak, set it to 0 so we start displaying it